west.yml

```diff
@@ -100,7 +100,7 @@ manifest:
       revision: f49bd1354616fae4093bf36e5eaee43c51a55127
       path: tools/net-tools
     - name: hal_nxp
-      revision: 27a464e4885f393dc376a53d9fc8d52078848496
+      revision: pull/88/head
```

Comment:
The hal PR has now been merged, so you can update this to the new SHA

Reply:
Done